hubspot/connection/__init__.py

```diff
@@ -204,6 +204,7 @@ def _require_successful_response(response):
             raise exception_class(
                 error_data['message'],
                 error_data['requestId'],
+                error_data,
```

Comment:
We are explicitly including only the data that we need in the _HUBSPOT_ERROR_RESPONSE_SCHEMA schema variable, since this commit is about adding more error data, it should be included in the schema so it can be validated.